core/include/robot_specs.h

```diff
@@ -7,8 +7,8 @@
  * that require info about the robot.
  * All distance measurements are in inches.
  */
-typedef struct 
-{
+typedef struct robot_specs_s

```

Comment:
I'm gonna be super picky and ask that we get rid of `robot_specs_s` to keep the consistency between struct definitions